docs/diagrams/SortCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,68 @@
+@startuml
+!include ./style.puml
+
+box "Logic" LOGIC_COLOR_T1
+
+Participant ":LogicManager" as logic LOGIC_COLOR
+Participant ":ReeveParser" as parser LOGIC_COLOR
+Participant ":SortCommandParser" as commandParser LOGIC_COLOR
+Participant "command:SortCommand" as command LOGIC_COLOR
+Participant "nameComparator:NameComparator" as nameComparator LOGIC_COLOR
+Participant ":ClassTimeComparator" as classTimeComparator LOGIC_COLOR
+Participant ":YearComparator" as yearComparator LOGIC_COLOR
+Participant ":CommandResult" as result LOGIC_COLOR
+
+[-> logic : execute()
+activate logic
+logic -> parser ++: parseCommand(commandText)
+create commandParser
+parser -> commandParser
+parser -> commandParser ++: parse(args)
+create command
+commandParser -> command ++: args
+return command
+return command
+return command
+destroy commandParser
+
+
+logic -> command ++: execute(model)
+
+endbox
+box "Model" MODEL_COLOR_T1
+Participant "model:Model" as model MODEL_COLOR
+
+opt If comparison means of command matches comparison means of Name Comparator
+
+    create nameComparator
+    command -> nameComparator ++:
+    return nameComparator
+    command -> model: updateSortedStudentList(nameComparator)
+    end
+
+opt If comparison means of command matches comparison means of Class Time Comparator
+
+    create classTimeComparator
+    command -> classTimeComparator ++:
+    return classTimeComparator
+    command -> model: updateSortedStudentList(classTimeComparator)
+    end
+
+opt If comparison means of command matches comparison means of Year Comparator
+
+    create yearComparator
+    command -> yearComparator ++:
+    return yearComparator
+    command -> model: updateSortedStudentList(yearComparator)
+    end
```

Comment:
Is it possible to just show `updateSortedStudentList(comparator)` instead of all these optional frames?

As long as your description in the DG is explains this part we can probably do away with these frames